### PR TITLE
Adding directions for install on a simple Ubuntu 20.04 machine

### PR DESCRIPTION
**Pull request by shamlian**
Removed a lot of information specific to Raspberry Pi and tested these directions with a VM on Windows. Short of OR improving their "not-Ubuntu" ROS 2 support, I think this will open doors for a lot of users.

### CODE REVIEW

File: docs/setup/ubuntu2004.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        source ~/.bashrc
+
+1. At this point, we recommend setting your default RMW. The RMW you set here has to match the RMW on your robot, which can be found from its Application Configuration page. More detail on RMW can be found [here](../xml-config). Right now, the Create® 3 robot supports `rmw_cyclonedds_cpp` and `rmw_fastrtps_cpp`. The default for Galactic is `rmw_cyclonedds_cpp`. Depending on your robot's RMW implementation, type one of the following:
+
+        echo "export RMW_IMPLEMENTATION=rmw_cyclonedds_cpp" >> ~/.bashrc
+or
+
+        echo "export RMW_IMPLEMENTATION=rmw_fastrtps_cpp" >> ~/.bashrc
+
+1. If both your computer and robot are on the same network, you should now be able to test things out with a `ros2 topic list`.
```
Comment (justinIRBT) [inline docs/setup/ubuntu2004.md:39]: Should we reference the xml-config.md that has more tips if the initial ros2 topic list fails?

File: docs/setup/ubuntu2004.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Install ROS 2 Galactic with Create 3 Messages on an Ubuntu 20.04 Machine
+
+## Before you start
+If you are running Ubuntu 20.04 natively on your machine, there is no extra setup required.
+These directions should work in a virtualized container within another operating system, as well.
+Note that there might be some network setup required if in a virtualized container; for example, RMWs seem to like running in a bridged network configuration rather than a NATted one.
```
Comment (justinIRBT) [inline docs/setup/ubuntu2004.md:6]: Should we give an example of what to do to setup a bridged network for a container if you have that info handy

Comment (shamlian) [reply docs/setup/ubuntu2004.md:6]: This is VM-software-dependent. For example, in VirtualBox, there's a setting in the "network" panel. I would have to try VMWare and Parallels and others to give a good answer.

File: docs/setup/ubuntu2004.md (hunk 1 above)
Comment (shamlian) [reply docs/setup/ubuntu2004.md:39]: Good idea.

File: docs/setup/ubuntu2004.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+## Step-by-step
+
+1. If you haven't already, download and install [Ubuntu® Server 20.04 64-bit](https://releases.ubuntu.com/20.04/ubuntu-20.04.4-live-server-amd64.iso)[^1] onto your machine.
+
+1. Once logged in, check to ensure that you are using a UTF-8 locale by typing
+
+        echo $LANG
+   and ensuring "UTF-8" is at the end of the returned string.
+
+1. Execute the following commands to install ROS 2[^2]:
```
[the remark this answers is not in the document]
Comment (shamlian) [reply docs/setup/ubuntu2004.md:17]: I almost did and then didn't. I will do that.

**Review by justinIRBT — APPROVED** [review]

**Pull request merged**